### ISSUE TITLE
настройка размера текста

### ISSUE THREAD

**Issue by Sergei-Litvinov** [tigran123/urantia-book-explorer#71]
при изменении размера текста в настройках не запоминается место, где открыт активный текст, поэтому приходится искать его снова

**Comment by Sergei-Litvinov** [comment]
изменяется размер текста главы, но не меняется размер текста для документа

**Comment by tigran123** [comment]
При изменении размера текста происходит перезагрузка всего сайта, т.е. эта проблема более глобальная. Соответственно, я изменю заголовок.

**Comment by tigran123** [comment]
Второе замечание я не понял: текст документа должен меняться, если ты его изменил в настройках. Ты уверен, что он не изменился. Этого не может быть и я не могу воспроизвести проблему отсюда.

**Comment by tigran123** [comment]
Я добавил запоминание того, какой столбец был активным, но это конечно ещё не решение. Нужно ещё запоминать какое _место_ в тексте было (и в активном и во всех остальных столбцах).

**Comment by Sergei-Litvinov** [comment]
не изменяется размер шрифта для **названия** документа, который расположен **в заголовке окна**

**Comment by tigran123** [comment]
Да, это действительно так, ибо и размер и даже тип шрифта для названия используются совсем иные, чем для текста документа. Но, я конечно могу добавить опцию в Настройки, которая позволит регулировать размер шрифта для названия документа в заголовке. Я создал проблему #73 для решения этой задачи.

**Comment by tigran123** [comment]
Проблема описанная в этом вопросе исчезла (ибо при изменении размера шрифта больше не требуется перезагрузка всего сайта), но вопрос я не закрываю, так как будет весьма полезно, если положение в тексте будет действительно сохраняться между сессиями.
